### PR TITLE
Add CN global_conf

### LINKED ISSUE #20: CN470-510 configuration
In China, there are two bands specified: CN470-510 and CN779-787. The TX power of the latter is limited to 10 dBm, which is too low.

Therefore, we are planning a CN470-510 configuration. Most channels are reserved for China Electric, so we need to select a frequency sub band (FSB) of the channels:
- 80-87
- 88-95

We are calling for contributions on defining the channels and testing.

### PR DESCRIPTION
**Pull request by htdvisser**
Resolves #20

### CODE REVIEW

File: CN-global_conf.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+		"lorawan_public": true,
+		"clksrc": 1,
+		"clksrc_desc": "radio_1 provides clock to concentrator",
+		"antenna_gain": 0,
+		"antenna_gain_desc": "antenna gain, in dBi",
+		"radio_0": {
+			"enable": true,
+			"type": "SX1255",
+			"freq": 486600000,
+			"rssi_offset": -176.0,
```
Comment (htdvisser) [inline CN-global_conf.json:12]: @tftelkamp: this is `-166.0` in the other configs, is this change ok?

File: CN-global_conf.json [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+			"tx_enable": true,
+			"tx_freq_min": 506700000,
+			"tx_freq_max": 508100000
+		},
+		"radio_1": {
+			"enable": true,
+			"type": "SX1255",
+			"freq": 487400000,
+			"rssi_offset": -176.0,
```
Comment (htdvisser) [inline CN-global_conf.json:21]: @tftelkamp: this is `-166.0` in the other configs, is this change ok?

Comment (tftelkamp) [reply CN-global_conf.json:21]: This number should be provided by the gateway manufacturer. I don't know what a good default is for SX1255 based gateways.

**Review by jpmeijers — COMMENTED** [review]
I don't see anything obviously wrong with this config. The most important part is that we have the frequencies and TX powers correct. What the other parameters are concerned, let's leave them as is until someone comments on it.

The frequencies seem correct to me.
I can't comment on the TX powers yet. Will do a bit of investigation on it.

**Comment by RimeLink** [comment]
We think the TxFreq shoud modified as:
"tx_freq_min": 500300000
"tx_freq_max":509700000

otherwise the gateway maybe REJECT emit packet of the RX2 that on 505.3MHz

**Comment by jpmeijers** [comment]
I agree that we should change the TX frequency limits to @RimeLink's suggestion. It is up to the network to decide which downlink frequency should be used. These limits should only prevent transmissions on out of band frequencies.
```
"tx_freq_min": 500300000
"tx_freq_max": 509700000
```

**Comment by jpmeijers** [comment]
Looking at this again:
"The LoRaWAN can be used in the Chinese 470-510MHz band..." - LoRaWAN Regional Parameters
From this we should perhaps use:
```
"tx_freq_min": 470000000
"tx_freq_max": 510000000
```
Then it will be inline with what the other configs do - allowing transmissions anywhere in the allowed band.

**Pull request merged**